site/content/contribute/getting-started/code-review.md

```diff
@@ -38,47 +38,65 @@ If you are a core committer seeking a review
+    * If your changes do not affect the user experience, you may remove `1: PM Review` immediately.
+6. After PM review, request a review from two [core committers](/contribute/getting-started/core-committers/).
+    * The choice of core committers is up to you.
+        - When picking your first core committer, consider someone with domain expertise relative to your changes. Sometimes GitHub will recommend a recent editor of the code, but often you must rely on your own intuition from past interactions.
+        - When picking your second core committer, consider someone from another team who may have expertise in the language you're using or the problem you're solving, even if they aren't intricately familiar with the codebase. This can provide a fresh set of eyes on the code to reveal blindspots that are not biased by hitting deadlines, and helps expose the team to new parts of the code to help spread out domain knowledge. This may not make sense for every pull request but is a practice to keep in mind.
+        - Don't be afraid to pick someone who gives "hard" reviews. Code review feedback is never a personal attack: it should "sharpen" the skills of both the author and the reviewers, not to mention improving the quality of the product.
+        - Try to avoid assigning the same person to all of your reviews unless they are related.
+        - When in doubt, ask for recommendations on our community server.
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing so as to avoid churn if changes are requested.
+    * Remove the `2: Dev Review` label only when these reviews are done.
+        - The last developer to review may choose to help here and speed up the process.
+7. After Dev review, assign a [QA tester](/contribute/getting-started/core-committers/#qa-testers).
+    * The choice of QA tester is up to you.
+        - In most cases, choose the QA tester embedded with your team.
+        - If your change primarily touches another team's codebase, consider their QA tester.
+        - If in doubt, assign any QA tester and comment about the uncertainty. The QA tester may reassign as appropriate.
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing.
+        - It is the QA tester's responsibility to determine the scope of required testing, if any.
+    * Remove the `3: QA Review` label only when their review is done.
+        - This is typically the responsibility of the QA tester.
+        - The QA tester may add the `QA Review Done` label when their review is done, or if a review is deemed not necessary.
+    * In the rare event your changes do not require QA review, you may remove the `3: QA Review` label yourself.
+        - Comment on the pull request clearly explaining the rationale.
+8. Merge the pull request.
+    * Do not merge until the `1: PM Review`, `2: Dev Review` and `3: QA Review` labels have been removed.
```

Comment:
"labels have been removed" - can they be removed by somebody else than the author? If no, maybe we can rephrase this to smth. like "you have removed", to be more explicit.